### PR TITLE
Migrations: Gitlab Add Reactions Support for Issues & MergeRequests

### PR DESCRIPTION
**Pull request by 6543**
as title

include  #12693

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

File: modules/migrations/gitlab.go [hunk 1]
```diff
@@ -384,6 +400,7 @@ func (g *GitlabDownloader) GetIssues(page, perPage int) ([]*base.Issue, bool, er
 }
 
 // GetComments returns comments according issueNumber
+// TODO: figure out how to transfer comment reactions
```
Comment (lunny) [inline modules/migrations/gitlab.go:403]: Remove this line since line 521?

**Review by lunny — APPROVED** [review]
LGTM except for https://github.com/go-gitea/gitea/pull/12695/files#r482700319

File: modules/migrations/gitlab.go (hunk 1 above)
Comment (lafriks) [inline modules/migrations/gitlab.go:403]:
```suggestion
```

Comment (lafriks) [reply modules/migrations/gitlab.go:403]: that is fo MRs

**Review by lafriks — APPROVED** [review]

**Pull request merged**